src/app/api/auth/login/callback/route.ts

```diff
@@ -40,19 +40,63 @@ export async function GET(request: NextRequest) {
       ssoProtocol: SSO_PROTOCOL as "openid-connect",
     })
 
-    // Create response with tokens (excluding refresh_token)
-    const responseData = {
-      access_token: tokens.access_token,
-      id_token: tokens.id_token,
-      expires_in: tokens.expires_in,
-      refresh_expires_in: tokens.refresh_expires_in,
-    }
+    // Create HTML response for successful login
+    const htmlContent = `
+      <!DOCTYPE html>
+      <html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Login Successful</title>
+        <style>
+          body {
+            font-family: Arial, sans-serif;
```

Comment:
Note: not using BCSans font as its not available in this context. We could add it but its a lot of bloat for a page that should automatically close almost instantly.